test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -808,6 +811,1175 @@ public void ProjectReferenceBetweenTwoGeneratingProjects(bool internalsVisibleTo
+                    (typeof(SourceGenerator), "SECURITY_ATTRIBUTES.g.cs", @"// ------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+// ------------------------------------------------------------------------------
+
+#pragma warning disable CS1591,CS1573,CS0465,CS0649,CS8019,CS1570,CS1584,CS1658
+namespace Windows.Win32
+{
+	using global::System;
+	using global::System.Diagnostics;
+	using global::System.Runtime.CompilerServices;
+	using global::System.Runtime.InteropServices;
+	using win32 = global::Windows.Win32;
+
+	namespace Security
+	{
+		/// <summary>The SECURITY_ATTRIBUTES structure contains the security descriptor for an object and specifies whether the handle retrieved by specifying this structure is inheritable.</summary>
+		/// <remarks>
+		/// <para><see href=""https://docs.microsoft.com/windows/win32/api//wtypesbase/ns-wtypesbase-security_attributes#"">Read more on docs.microsoft.com</see>.</para>
+		/// </remarks>
+		internal partial struct SECURITY_ATTRIBUTES
+		{
+			/// <summary>The size, in bytes, of this structure. Set this value to the size of the **SECURITY\_ATTRIBUTES** structure.</summary>
+			internal uint nLength;
+			/// <summary>
+			/// <para>A pointer to a [**SECURITY\_DESCRIPTOR**](../winnt/ns-winnt-security_descriptor.md) structure that controls access to the object. If the value of this member is **NULL**, the object is assigned the default security descriptor associated with the [*access token*](/windows/win32/secauthz/access-tokens) of the calling process. This is not the same as granting access to everyone by assigning a **NULL** [*discretionary access control list*](/windows/win32/secauthz/dacls-and-aces) (DACL). By default, the default DACL in the access token of a process allows access only to the user represented by the access token. For information about creating a security descriptor, see [Creating a Security Descriptor](/windows/win32/secauthz/creating-a-security-descriptor-for-a-new-object-in-c--).</para>
+			/// <para><see href=""https://docs.microsoft.com/windows/win32/api//wtypesbase/ns-wtypesbase-security_attributes#members"">Read more on docs.microsoft.com</see>.</para>
+			/// </summary>
+			internal unsafe void* lpSecurityDescriptor;
+			/// <summary>A Boolean value that specifies whether the returned handle is inherited when a new process is created. If this member is **TRUE**, the new process inherits the handle.</summary>
+			internal win32.Foundation.BOOL bInheritHandle;
+		}
+	}
+}
+".Replace("\r\n", "\n")),
+                },
+            },
+        }.RunAsync();
+    }
+
     private static ImmutableArray<Diagnostic> FilterDiagnostics(ImmutableArray<Diagnostic> diagnostics) => diagnostics.Where(d => d.Severity > DiagnosticSeverity.Hidden).ToImmutableArray();
 
```

Comment:
Find this line and remove it, since you're now demonstrating a way.

```cs
// CONSIDER: How can I pass in the source generator itself, with AdditionalFiles, so I'm exercising that code too?
```